aws/resources/nat_gateway.go

```diff
@@ -183,3 +181,15 @@ type TooManyNatErr struct{}
 func (err TooManyNatErr) Error() string {
 	return "Too many NAT Gateways requested at once."
 }
+
+func nukeNATGateway(client ec2iface.EC2API, gateway *string) error {
+	logging.Debugf("Deleting NAT gateway %s", awsgo.StringValue(gateway))
+
+	_, err := client.DeleteNatGateway(&ec2.DeleteNatGatewayInput{NatGatewayId: gateway})
+	if err != nil {
+		logging.Debugf("[Failed] Error deleting NAT gateway %s: %s", awsgo.StringValue(gateway), err)
+		return errors.WithStackTrace(err)
+	}
+	logging.Debugf("[Ok] NAT Gateway deleted successfully %s", awsgo.StringValue(gateway))
```

Comment:
Some debug statements have `[Ok]` and `[Failed]`, but some don't. Is this intentional?

Reply:
Not really. Haha I can look into making the log consistent in a follow-up PR.